Makefile

```diff
@@ -1,11 +1,8 @@
-.PHONY: install run clean
-
-install:
-	pipenv install
+.PHONY: clean
 
 clean:
-	rm -rf src/backend/*.egg-info
-	find . -type f -name \*~ | xargs rm
+	rm -rf src/backend/*.egg-info &
+	find . -type f -name \*~ | xargs rm &
 	find . -type f -name \*pyc | xargs rm
 
 IMAGES := database_image expungeservice_image webserver_image
```

Comment:
I don't see that this variable is referenced anywhere. Is it still needed?

Reply:
You're right, I can remove it.